python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -1380,17 +1380,12 @@ def fillna(
                 )
             return getattr(self, method, limit)()
 
-        value_columns = self.grouping.values
-        _, (data, index), _ = self._groupby.groups(
-            cudf.core.frame.Frame(value_columns._data)
+        values = self.obj.__class__._from_data(
+            self.grouping.values._data, self.obj.index
         )
-
-        grouped = self.obj.__class__._from_data(data, index)
-        result = grouped.fillna(
+        return values.fillna(
```

Comment:
Hey! Here's an untested code path found by coverage! Can we add a test? This looks like the code that gets hit if we do a `fillna` without specifying a method.

Reply:
The annotation disappeared. Closing for now. :shrug: